// kapellmeister/types.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "now", {
  enumerable: true,
  get: function get() {
    return _d3Timer.now;
  }
});
Object.defineProperty(exports, "Timer", {
  enumerable: true,
  get: function get() {
    return _d3Timer.Timer;
  }
});
Object.defineProperty(exports, "timer", {
  enumerable: true,
  get: function get() {
    return _d3Timer.timer;
  }
});
Object.defineProperty(exports, "interval", {
  enumerable: true,
  get: function get() {
    return _d3Timer.interval;
  }
});
Object.defineProperty(exports, "timeout", {
  enumerable: true,
  get: function get() {
    return _d3Timer.timeout;
  }
});
Object.defineProperty(exports, "BaseNode", {
  enumerable: true,
  get: function get() {
    return _BaseNode.default;
  }
});

var _d3Timer = require("d3-timer");

var _BaseNode = _interopRequireDefault(require("./BaseNode"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
